persistence/test/setup_test.go

```diff
@@ -39,7 +39,7 @@ var (
 	DefaultMaxRelays     = types.BigIntToString(DefaultMaxRelaysBig)
 	StakeToUpdate        = types.BigIntToString((&big.Int{}).Add(DefaultStakeBig, DefaultDeltaBig))
 
-	DefaultStakeStatus     = persistence.StakedStatus
+	DefaultStakeStatus     = int32(persistence.StakedStatus)
```

Comment:
`StakedStatus` is just an int where it is defined.

```
// IMPROVE(team): Move this into a proto enum. We are not using `iota` for the time being
// for the purpose of being explicit: https://github.com/pokt-network/pocket/pull/140#discussion_r939731342
// TODO Cleanup with #149
const (
	UndefinedStakingStatus = 0
	UnstakingStatus        = 1
	StakedStatus           = 2
	UnstakedStatus         = 3
)
```

We should do one of two things:
1. Create a staking enum status in persistence (it'll be a mirror of what we have in utility, but also keeps it separate and independent in case it needs to change in the future).
2. Initialize the constants as `int32`.

My preference is (1) but lmk what you think

Reply:
I think 1 makes sense, but ultimately they shouldn't even be sharing `enums`

Reply:
Staked status is going to go away anyway